### DISCUSSION TITLE
Separated Folder/Album view in the app

### DISCUSSION THREAD

**Discussion by dfoxg** [Radiokot/photoprism-android-client#69] [Ideas]
In my opinion it would be great if the app would have a [navigation bar](https://m3.material.io/components/navigation-bar/overview) in the footer with following elements:
- Gallery (the current view)
- Albums/Folders (a [card-view](https://m3.material.io/components/cards/overview) with all the albums or folders photoprism has. After selection one of the album, the view should display all photos and videos in that album/folder. When the user use the `back gesture` the view should be go back into album/folders overview.
- Settings (The current view if you click on the three dots in the top left corner)

**Comment by Radiokot** [comment]
Thank you for the detailed description. What do you think is the difference between viewing the album/folder content from a specific tab rather than from the search screen?

**Reply by dfoxg** [reply]
I have a lot of albums (over 200) and at least for me, finding the correct album with the current depiction is quite annoying.

**Reply by dfoxg** [reply]
I think the root cause is, that all of my albums are in one line, rather then a standard (searchable) [list](https://m3.material.io/components/lists/overview)

**Reply by Radiokot** [reply]
The horizontal list is really useless for such a big number of albums. Should have a better view for that. For now, you can star your most used albums and folders in the web app – they will always be at the beginning of the list.

**Reply by Radiokot** [reply]
Hi Daniel. Check out the latest release:
https://github.com/Radiokot/photoprism-android-client/releases/tag/1.17.0(27)
Now the app has albums overview screen, where you can see all the albums in grid view, quickly scroll from top to bottom and filter the items by name.

**Reply by dfoxg** [reply]
Hi Oleg, thanks a lot, that will really helps well! I love the search feature 🫶

A little detail: what is the current order-factor? Maybe it would be good if the album's are ordered by name?

Another Idea: can you do exactly the same with die people chooser?

**Reply by dfoxg** [reply]
One last thing: The People people chooser has multiselect, the folder chooser hasn't. Is this a bug due a technical limitations or a feature?

**Reply by Radiokot** [reply]
In the albums list, favorite albums and folders go first, and the rest follow them, ordered by the last update time. In the people list, favorite named people go first, and the new unnamed faces follow them.

💡 To make some album, folder or person always appear at the beginning of the albums list, go to the PhotoPrism web app and star the item.

I've added multiselect to the people filter because it looked reasonable. Such galleries as Google Photos and Samsung gallery allow selecting multiple faces and get photos with all of them. As for albums and folders, only a single selection is available because naturally you can't open multiple folders as one, although PhotoPrism query supports combining albums with `albums` keyword, like `albums:August|Novebmer`

**Reply by dfoxg** [reply]
Got it. At least in my opinion the current order is quite confusing, I think a sorting by name should be the default, since it's the same in the photoprism web client. But that's only my opinion:)

It would be great if you can add the overview screen to the people chooser too :)

**Reply by Radiokot** [reply]
I agree, the relevance sorting seems inconsistent as the items move around the list sometimes. Sort by name and overview screen for people will be added soon.

**Reply by kamiyaa** [reply]
I would also like a separate page for albums.
When I have thousands of albums, scrolling through them in a single row search is not feasible.
Would be much better to have a list of albums that can be scrolled and filtered through.

**Reply by Radiokot** [reply]
@kamiyaa it is already available in the latest version
